src/pybind/mgr/mgr_util.py

```diff
@@ -1,5 +1,8 @@
 import os
 
+from ceph.fs.earmarking import CephFSVolumeEarmarking, \
+    EarmarkException, EarmarkTopScope
```

Comment:
nitpick: use the preferred line-continuation-with-parens style
```suggestion
from ceph.fs.earmarking import (
    CephFSVolumeEarmarking,
    EarmarkTopScope,
    EarmarkException
)
```

From pep8:
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.